examples/SnapML/README.md

```diff
@@ -9,3 +9,8 @@ The [nyc_taxi_tip_predict_notebook.ipynb](nyc_taxi_tip_predict_notebook.ipynb) n
 demonstrates training a Ridge Regression and Logistic Regression model using Snap ML.
 The models trained using Snap ML are compared for accuracy and performance against scikit-learn
 and RAPIDS cuML.
+
+## Allstate Claim prediction challenge using Snap ML Logistic Regression
```

Comment:
I'd probably add the Kaggle link to the README here as well. That would clarify that the use of "Allstate" here is in reference to the existing challenge name, rather than implying some special relationship or agreement between IBM / PowerAI and Allstate.

Reply:
Thanks for the review @hartb. I have fixed the README. Thanks!